h/static/scripts/test/root-thread-test.js

```diff
@@ -212,6 +212,19 @@ describe('rootThread', function () {
       assert.isTrue(threadFilterFn(fakeBuildThread));
     });
 
+    it('generates a thread filter function to match orphans', function () {
+      fakeBuildThread.reset();
+      fakeBuildThread.annotation = {$orphan: true};
+
+      fakeAnnotationUI.state = Object.assign({}, fakeAnnotationUI.state,
+        {selectedTab: 'orphan'});
```

Comment:
This should use the constant for the orphans tab rather than the string literal.